runtime/gc_base/IndexableObjectAllocationModel.cpp

```diff
@@ -287,5 +316,78 @@ MM_IndexableObjectAllocationModel::layoutDiscontiguousArraylet(MM_EnvironmentBas
 	return spine;
 }
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+#if !defined(LINUX) || !defined(J9VM_ENV_DATA64) || defined(OMRZTPF)
```

Comment:
I would change this for easier read:
`#if !(defined(LINUX) && defined(J9VM_ENV_DATA64)) || defined(OMRZTPF)`